prac_10/flaskProject.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():  # put application's code here
+    return "<h1>Hello World :)</h1>"
+
+
+@app.route('/greet')
+@app.route('/greet/<name>')
+def greet(name=""):
+    return f"Hello {name}"
+
+
+@app.route('/f/<celsius>')
+def calculate_fahrenheit_from_celsius(celsius):
```

Comment:
I think you need to write "celsius= 22(as an example)"

Reply:
Noted. Thank you.